src/Juvix/Data/Fixity.hs

```diff
@@ -2,21 +2,15 @@ module Juvix.Data.Fixity where
 
 import Juvix.Prelude.Base
 
+-- | Note that the order of the constructors is important due to the `Ord`
+-- instance.
+-- TODO should we rename PrecMinusOmega to PrecApp, PrecMinusOmega1 to PrecUpdate, and PrecOmega to PrecFunction/Arrow?
```

Comment:
I think the renaming would be a good idea. Do we use these precedences for anything other than application, update and arrow?

Reply:
> Do we use these precedences for anything other than application, update and arrow?

I think that is an exhaustive list, so no.
Maybe you can do apply change in #2270.

Reply:
Yes, makes sense, I can do the renaming there. Though not to disrupt the code too much I'm not getting rid of the old Fixity/Precedence framework, but translating the precedence graph to a linear precedence ordering at scoping.